app/mailers/mailer.rb

```diff
@@ -12,16 +13,18 @@ def email_test(destination_address)
     @time = Time.current
     mail(:to => destination_address, :subject => 'Testing')
   end
-  
-  def confirm_account_change(customer, whathappened, newpass=nil)
+
+  def confirm_account_change(customer, whathappened, token='', requestURL)
     @whathappened = whathappened
-    @newpass = newpass
+    uri = URI(requestURL)
+    @tokenLink = "#{uri.scheme}://#{uri.host}" + "/customers/reset_token?token=" + token
     @customer = customer
+    puts("The reset password token link is: #{@tokenLink}")
     mail(:to => customer.email, :subject => "#{@subject} #{customer.full_name}'s account")
   end
 
-    
-  def confirm_order(purchaser,order) 
+
```

Comment:
This is minor, but when doing PRs upstream, please try to avoid putting in extra blank lines.  They show up as extra diffs that require cognitive load on the part of the maintainer to scan.  Try to really aim for the absolute minimum number of changes to make your code work.  It may be that the extra blank lines are pleasing to you, but if they're extraneous, they just create work for the maintainer.